api-spec/protobuf/ark/v1/service.proto

```diff
@@ -189,6 +189,7 @@ message GetInfoResponse {
   string network = 5;
   int64 dust = 6;
   string boarding_descriptor_template = 7;
+  string forfeit_address = 8;
```

Comment:
Worth using descriptor also to be consistent? As we already have the network as 5th.

Reply:
I don't see any advantage to use descriptor, there is no "arguments" and the user doesn't have to generate new addresses from it. Also, `forfeit_address` could be any type of address, so it means supporting all type of descriptors etc... sounds a bit overkill to use descriptor only for a recipient segwit v0 address

Reply:
also, an address can be easily copy/pasted in any explorer to verify if ASP puts some forfeits transactions onchain

Reply:
What if, instead of this and pubkey, we returned the descriptor that specifies how the forfeit address is  done and encodes also the pubkey?

Reply:
public key = key of the ASP to use in VTXO script
forfeit address = ASP script to use when building the forfeits

the forfeit address / script is just a way to agree on a common script while adding the ASP output in forfeits. it does not have to contain the ASP public key. In fact that's not the case now, forfeit address is the first main account address (a segwit v0 account, not the one used to compute ASP public key)

forfeit address is a wallet interface address. First, none of the wallets we are using are descriptor based.
 And if I see the advantages of descriptors in case of taproot VTXO scripts cause it contains arguments etc... I'm not sure why we should use a descriptor to encode a classic p2wpkh address (also because our descriptor package only supports taproot descriptor so it means extra work there to just communicate an output script)